instructionAPI/src/Operand.C

```diff
@@ -134,6 +134,21 @@ namespace Dyninst { namespace InstructionAPI {
       s.replace(0, 2, "0x0("); // fix-up ##X to indirection syntax 0x0(X)
       s += ')';
     }
+    if(this->isRead() || this->isWritten()) {
+      s += "{";
+      bool needs_comma = false;
```

Comment:
I guess the goal is to generate either {r}, {w} or {r,w}, but maybe worth commenting.